### PR TITLE
Test: use locator for textContent, hover, fill and getAttribute

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Test: use locator for textContent, hover and getAttribute
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | ALL :green_circle: 
| UI Tests          | https://github.com/boubkerbribri/ga.tests.ui.pr/actions/runs/7195636480
| Fixed issue or discussion?     | N/A
| Related PRs       | N/A
| Sponsor company   | PrestaShopCorp

### CODE REVIEW

**Review by Progi1984 — DISMISSED** [review]

File: tests/UI/campaigns/modules/ps_apiresources/01_installation/02_disableEnableModule.ts [hunk 1]
```diff
@@ -123,6 +123,7 @@ describe('PrestaShop API Resources module - Disable/Enable module', async () =>
         await testContext.addContextItem(this, 'testIdentifier', `checkScopesCore${index}`, baseContext);
 
         const scopes = await addNewApiAccessPage.getApiScopes(page, '__core_scopes');
+        console.log(scopes);
```
Comment (Progi1984) [inline tests/UI/campaigns/modules/ps_apiresources/01_installation/02_disableEnableModule.ts]: @boubkerbribri To remove

**Review by Progi1984 — COMMENTED** [review]
A feedback

File: tests/UI/campaigns/modules/ps_apiresources/01_installation/02_disableEnableModule.ts (hunk 1 above)
Comment (boubkerbribri) [reply tests/UI/campaigns/modules/ps_apiresources/01_installation/02_disableEnableModule.ts]:
```suggestion
```

**Review by Progi1984 — APPROVED** [review]

**Review by nicosomb — APPROVED** [review]
ok for the vue file

**Review by matthieu-rolland — APPROVED** [review]

**Comment by jolelievre** [comment]
Thanks @boubkerbribri

**Pull request merged**